services/mgmt/cdn/src/package_2019_12/models.rs

```diff
@@ -266,7 +266,7 @@ pub mod remote_address_match_condition_parameters {
     #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
     pub enum Odata_type {
         #[serde(rename = "#Microsoft.Azure.Cdn.Models.DeliveryRuleRemoteAddressConditionParameters")]
-        MicrosoftAzureCdnModelsDeliveryRuleRemoteAddressConditionParameters,
+        U23MicrosoftAzureCdnModelsDeliveryRuleRemoteAddressConditionParameters,
```

Comment:
Here, the first character is a number sign, so it gets replace with `U23`, the unicode hex value. https://www.unicodepedia.com/unicode/basic-latin/23/number-sign/